src_c/rwobject.c

```diff
@@ -536,7 +536,7 @@ _pg_rw_seek(SDL_RWops *context, Sint64 offset, int whence)
     PyGILState_STATE state;
 
     if (helper->fileno != -1) {
-        return lseek(helper->fileno, offset, whence);
+        return lseek(helper->fileno, (long)offset, whence);
```

Comment:
Ah. This is an error. We should be using `lseek64(helper->fileno, (long)offset, whence)`

Reply:
Oh right, nice catch!